src/status/status.factory.js

```diff
@@ -0,0 +1,209 @@
+const { stripIndent } = require('common-tags');
+
+
+module.exports = (logProvider, stateProvider, logger) => {
+    //Helper functions
+    function activeMembers(events) {
+        for(var i=0; i < event.length; i++){
+            var obj = events.logs[i];
+                if(obj.action == "Participated"){
+                    return ({obj})
+                }
+            }
+    }
+
+    function activeEvents(events){
+     for(var i=0; i < events.team_id.length; i++){
+         var obj = events;
+         if(obj.halted == true){
+                return ({obj});
+            }
+        }
+    }
+
+    function happenedInPeriodUser(date, action, back){
+        const alerts = ({
+            notified:0,
+            missed:0
+        });
+        const settingDate = new Date();
+        settingDate.setDate(date);
+        if( back != 'all')
+        {
+            const hasHappened = settingDate - back;
+            while(hasHappened.getUTCDate() < settingDate.getUTCDate()){
+                if(action == "Participated") { alerts.notified =+ 1;}
```

Comment:
we record both "Participated" and "Notified" in the logs at the same time. So for the same event for the same user, there can be two logs. How is this code taking what I mentioned into account?